engine/src/main/scala/cromwell/jobstore/JobStoreService.scala

```diff
@@ -0,0 +1,57 @@
+package cromwell.jobstore
+
+import akka.actor.Actor
+import com.typesafe.config.Config
+import cromwell.core.WorkflowId
+import cromwell.jobstore.JobStoreService.{JobStoreReaderCommand, JobStoreWriterCommand}
+import cromwell.services.ServiceRegistryActor.ServiceRegistryMessage
+
+
+/**
+  * Joins the service registry API to the JobStoreReaderActor and JobStoreWriterActor.
+  *
+  * This level of indirection is a tiny bit awkward but allows the database to be injected.
+  */
+case class JobStoreService(serviceConfig: Config, globalConfig: Config) extends Actor {
+
+  // TODO: Replace with a real database, probably from config.
+  val database = FilesystemJobStoreDatabase
+  val jobStoreWriterActor = context.actorOf(JobStoreWriterActor.props(database))
+  val jobStoreReaderActor = context.actorOf(JobStoreReaderActor.props(database))
```

Comment:
TOL I could imagine this reader not being a singleton, i.e. one-per-request or router based... But that's probably a story for another summer